resources/conda_menu_config.json

```diff
@@ -18,7 +18,8 @@
             "platforms": {
                 "win": {
                     "precommand": "set \"PYTHONHOME=\" & set \"PYTHONPATH=\"",
-                    "desktop": true
+                    "desktop": true,
+                    "app_user_model_id": "napari.napari"
```

Comment:
```suggestion
                    "app_user_model_id": "napari.napari.viewer.__PKG_VERSION__"
```

Reply:
I don't know who is setting this internally (Qt?), but it uses the full version too, and they both need to match. Ideally the version wouldn't be set from the Qt side, but that's how it is now.